src/lib/components/side-nav.svelte

```diff
@@ -96,6 +106,25 @@
         <div class="nav-title">{translate('feedback')}</div>
       </NavRow>
     </slot>
+    <NavRow {isCloud} handleClick={() => ($labsMode = !$labsMode)}>
+      <NavTooltip right text={labsHoverText}>
+        <button
+          aria-label={labsText}
+          data-testid="labs-button"
+          class="w-10 flex items-center justify-center"
+        >
+          <Icon name="labs" active={$labsMode} />
+        </button>
+      </NavTooltip>
+      <div class="nav-title flex flex-col leading-3">
+        <div>{labsText}</div>
+        {#if $labsMode}
+          <p class="text-[12px]">
```

Comment:
```suggestion
          <p class="text-xs mt-1 font-normal">
```

| Before | After |
|--|--|
|![Screenshot 2023-08-10 at 11 22 20 AM](https://github.com/temporalio/ui/assets/15069288/015ffdf7-6b77-4526-a339-82c7f9289734)|![Screenshot 2023-08-10 at 11 22 04 AM](https://github.com/temporalio/ui/assets/15069288/16664ca0-7ea0-4ca0-8ab9-e70a4c9f3e22)|